src/test/java/nextstep/subway/line/LineDistanceAcceptanceTest.java

```diff
@@ -0,0 +1,98 @@
+package nextstep.subway.line;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.AcceptanceTest;
+import nextstep.subway.line.dto.LineRequest;
+import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.station.domain.Station;
+import org.junit.jupiter.api.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 노선 구간 조회 기능")
+public class LineDistanceAcceptanceTest extends AcceptanceTest {
+
+	@DisplayName("노선 생성 시 종점역(상행, 하행)을 함께 추가하기")
+	@Test
+	void createLineWithTerminal() throws JsonProcessingException {
+		String upStationName = "강변역";
+		String downStationName = "건대입구역";
+		Long upStationId = 역_생성_후_stationId응답(upStationName);
+		Long downStationId = 역_생성_후_stationId응답(downStationName);
+		// when
+		// 지하철_노선_생성_요청
+		String lineName = "2호선";
+		ExtractableResponse response = 노선_생성_함수(lineName, "bg-green-600", upStationId, downStationId, 3);
+		// then
+		// 지하철_노선_생성됨
+		assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+		String lineId = response.header("Location").split("/")[2];
+		ExtractableResponse<Response> linesResponse = ID로_노선을_조회한다(Long.valueOf(lineId));
+		assertThat(linesResponse.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+		LineResponse lineResponse = linesResponse.jsonPath().getObject(".", LineResponse.class);
+		assertThat(lineResponse.getName()).isEqualTo(lineName);
+		List<String> stationNames = lineResponse.getStations().stream().map(Station::getName).collect(Collectors.toList());
+		assertThat(stationNames.equals(Arrays.asList(upStationName, downStationName))).isTrue();
```

Comment:
assertj 에는 가독성을 높이기 위한 검증문을 많이 제공하니 이를 활용해봐도 좋을 것 같아요 :)

* https://github.com/assertj/assertj-examples/blob/main/assertions-examples/src/test/java/org/assertj/examples/IterableAssertionsExamples.java


검증문

```suggestion
		assertThat(stationNames).containsExactly(upStationName, downStationName);
```

혹은 검증 데이터 추출 및 검증

```suggestion
		assertThat(lineResponse.getStations())
		.extracting(Station::getName)
		.containsExactly(upStationName, downStationName);
```

Reply:
저런 함수도 있었네요 감삼다